tests/unit/sparql/QueryObjectBuilder.spec.ts

```diff
@@ -101,9 +101,216 @@ describe( 'QueryObjectBuilder', () => {
 			omitLabels: true,
 		};
 
-		expect( () => {
-			builder.buildFromQueryRepresentation( quantityQueryRepresentation );
-		} ).toThrow();
+		const expected = {
```

Comment:
This is massive, can you move it to a json file or another js script?

Reply:
And it was a pain to manually write, let me tell you that. I'm still looking into options here.

Reply:
:( let me know if you found an option.

Reply:
I extracted it into a .json file. What do you think?